app/main.py

```diff
@@ -2,0 +1,16 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for visitor in friends:
+        try:
+            cafe.visit_cafe(visitor)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+    if masks_to_buy:
+        return f"Friends should buy {masks_to_buy} masks"
+    return f"Friends can go to {cafe.name}"
```

Comment:
Code Style Issue: Avoid using unnecessary `else`. Here, you can simplify the code by removing the `else` statement and unindenting its block. This will make the code more readable and idiomatic.